src/compiler/types.ts

```diff
@@ -5691,9 +5691,12 @@ namespace ts {
 
     export interface TupleType extends GenericType {
         elementFlags: readonly ElementFlags[];
-        minLength: number;  // Number of required or variadic elements
-        fixedLength: number;  // Number of initial required or optional elements
-        hasRestElement: boolean;  // True if tuple has any rest or variadic elements
+        /** Number of required or variadic elements */
+        minLength: number;
+        /** Number of initial required or optional elements */
+        fixedLength: number;
+        /** True if tuple has any rest or variadic elements */
+        hasRestElement: boolean;

```

Comment:
By moving those to JSDocs we improve slightly the IDE experience as this information gets displayed in the tooltips when hovering over those properties